UltiSnips/elixir.snippets

```diff
@@ -0,0 +1,126 @@
+snippet if
+if ${1:condition} do
+	${2:expression}
+${3:end}
```

Comment:
I wasn't optimizing for keystrokes, here, but for a reduced number of things to remember. I think you have a point, though... If there was a potential `elsif` or similar, that be that would make more sense.

Is there a standard beyond "what's been done before in this repo"? Not that that's necessarily a poor standard, I'm just wondering if I'm missing a larger context or something.